examples/extras/event_study.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+import numpy as np
+import san
+
+import sys
+sys.path.append("../..")
+
+from san.extras.event_study import event_study, signals_format
+
+data = san.get("ohlcv/bitcoin")
+data["bitcoin"] = data.closePriceUsd
+data["ETH_close"] = san.get("ohlcv/ethereum").closePriceUsd
+data["daily_active_addresses_ETH"] = san.get("daily_active_addresses/ethereum")
+
+data["daa_performance"] = np.log(data["daily_active_addresses_ETH"].pct_change() + 1)
+data["sd_rolling"] = data["daa_performance"].rolling(100).std()
+
+price = data[["ETH_close", "bitcoin"]]
+price=price.tz_convert(None)  # Event Study needs date instead of datetime
+
+# Defining signals:
+# Buy Signal: Is the current trend lower than two standard deviations?
+data["buy"] = (data["daa_performance"] < (data["sd_rolling"] * (-1) * 2))
+# Sell Signal: Is the current trend higher than two standard deviations?
+data["sell"] = (data["daa_performance"] > (data["sd_rolling"] * 2))
+
+
+# Helper function to get the signals in the right format:
+signals = signals_format(data["sell"],'ETH_close')
+
+# Calling the event study:
+event_study(price, signals, starting_point = 30)
```

Comment:
The code here is not organized in functions/classes?

Reply:
This is not part of the module but the example on how to prepare signals and evaluate them via event study. Its meant to be run in a jupyter notebook. So I dont know if this needs to be modular. But I can do it of you say its more usable that way.

Reply:
May be this should be converted to a Jupiter notebook?

Reply:
Jup makes sense